modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/app/ApplicationConfig.scala

```diff
@@ -50,6 +54,25 @@ object ApplicationConfig {
     apiUrl  : Option[String]
   )
 
+  case class WalletsInfo(
+    address      : Address,
+    minimumAmount: Long,
+    rewardAmount : Long,
+    afterDate    : LocalDate
+  )
+
+  case class InflowTransactionsDaemonConfig(
+    idleTime   : FiniteDuration,
+    apiUrl     : Option[String],
```

Comment:
String allows you to pass anything, even not a url. It should be a newtype and also a refined type